commons/pom.xml

```diff
@@ -101,7 +101,7 @@
 		<dependency>
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-classic</artifactId>
-			<version>1.5.6</version>
+			<version>1.5.16</version>
 		</dependency>
 		<dependency>
 			<groupId>com.alibaba.fastjson2</groupId>
```

Comment:
This can be updated to 2.0.54 and updating HikariCP to 6.2.1 should be safe too.